### PR TITLE
[R4R]uprade to tendermint release 0.32.3

### PR DESCRIPTION
**Pull request by guagualvcha**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -2,30 +2,21 @@ module github.com/binance-chain/go-sdk
 
 require (
 	github.com/binance-chain/ledger-cosmos-go v0.9.9-binance.1
-	github.com/btcsuite/btcd v0.0.0-20181013004428-67e573d211ac
+	github.com/btcsuite/btcd v0.0.0-20190115013929-ed77733ec07d
```
Comment (HaoyangLiu) [inline go.mod]: Shall we keep it consistent with cosmos repo? We use `v0.20.0-beta` in cosmos-sdk

File: types/tx/stdtx.go [hunk 2]
```diff
@@ -35,11 +33,4 @@ func NewStdTx(msgs []msg.Msg, sigs []StdSignature, memo string, source int64, da
...
 // GetMsgs def
-func (tx StdTx) GetMsgs() []msg.Msg { return tx.Msgs }
-
-type Info struct {
-	Hash   common.HexBytes         `json:"hash"`
-	Height int64                   `json:"height"`
-	Tx     Tx                      `json:"tx"`
-	Result types.ResponseDeliverTx `json:"result"`
-}
+func (tx StdTx) GetMsgs() []msg.Msg { return tx.Msgs }
```
Comment (HaoyangLiu) [inline types/tx/stdtx.go]: format this

File: client/rpc/types.go [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+
+type ResponseCheckTx struct {
+	Code      uint32        `json:"code,omitempty"`
+	Data      []byte        `json:"data,omitempty"`
+	Log       string        `json:"log,omitempty"`
+	Info      string        `json:"info,omitempty"`
+	GasWanted int64         `json:"gas_wanted,omitempty"`
+	GasUsed   int64         `json:"gas_used,omitempty"`
+	Events    []types.Event `json:"events,omitempty"`
+	Tags      []cmn.KVPair  `json:"tags,omitempty"`
```
Comment (HaoyangLiu) [inline client/rpc/types.go]: Why there are both `Events` and `Tags`?

File: go.mod (hunk 1 above)
Comment (guagualvcha) [reply go.mod]: sure.

File: types/tx/stdtx.go (hunk 2 above)
Comment (guagualvcha) [reply types/tx/stdtx.go]: done

File: client/rpc/types.go (hunk 3 above)
Comment (guagualvcha) [reply client/rpc/types.go]: for  compatibility

File: e2e/e2e_rpc_test.go [hunk 4]
```diff
@@ -23,13 +23,13 @@ import (
 )
 
 var (
-	nodeAddr           = "tcp://127.0.0.1:80"
+	nodeAddr           = "tcp://data-seed-pre-0-s3.binance.org:80"
```
Comment (yutianwu) [inline e2e/e2e_rpc_test.go]: should we change node address to testnet node

**Pull request merged**